### PR TITLE
Clear ilicache button

### LINKED ISSUE #776: Button to clear ilicache
Lots of users struggle with the .ilicache. Either they are not aware or they don't know that there is a .ilicache which could cause some problems.

It would be fancy to have a "Clear ILI-Cache" Button in Model Baker, so users (a) know that there is a cache and (b) can empty the cache very easily without having to know how ili2db handles the cache.

E.g. like this:

image

### PR DESCRIPTION
**Pull request by domi4484**
Fix #776

Depends on https://github.com/opengisch/QgisModelBakerLibrary/pull/60

image

### CODE REVIEW

File: QgisModelBaker/gui/workflow_wizard/import_source_selection_page.py [hunk 1]
```diff
@@ -186,3 +189,6 @@ def _remove_selected_rows(self):
         indices = self.source_list_view.selectionModel().selectedIndexes()
         self.source_list_view.model().remove_sources(indices)
         self.remove_button.setEnabled(self._valid_selection())
+
+    def _clear_ilicache_button_clicked(self):
+        print("_clear_ilicache_button_clicked")
```
Comment (signedav) [inline QgisModelBaker/gui/workflow_wizard/import_source_selection_page.py:194]: And what happens then?

Comment (domi4484) [reply QgisModelBaker/gui/workflow_wizard/import_source_selection_page.py:194]: ouch I have lost a commit somewhere

**Review by signedav — APPROVED** [review]
The code looks good. Have you tested it out with the whole workflow?

**Comment by domi4484** [comment]
> The code looks good. Have you tested it out with the whole workflow?

I have checked that the cache directories get deleted and then created again  when running some ili operations again.

Is there something else I should pay attention?

**Pull request merged**